sdk/core/azure-core/src/main/resources/META-INF/native-image/com.azure/azure-core/native-image.properties

```diff
@@ -1,14 +1,22 @@
 Args=\
-  --allow-incomplete-classpath \
```

Comment:
why are we removing this property ?
is thing going to have any breaking impact ?

Reply:
This is the default behavior for all new versions of GraalVM and we are currently using the new version. So, this property is not required.